server/controllers/profile.js

```diff
@@ -0,0 +1,71 @@
+// We would like to create the following controllers for creating, updating or getting profiles. Include the routes for this ticket as well
+
+// - CREATE
+
+// - Given parameters passed in, create a profile.
+
+// - UPDATE
+
+// - Given an ID and new parameters, update the profile
+
+// - GET
+
+// - Given an ID, return profile with that ID
+
+// - GET all
+
+// - A list of profiles
+const Profile = require("../models/Profile");
+const asyncHandler = require("express-async-handler");
+
+
+exports.searchProfiles = asyncHandler(async (req, res, next) => {
+
+  const profiles = await Profile.find({});
+
+  res.status(200).json({ profiles: profiles });
+});
```

Comment:
Search denotes the ability to pass in data and return potential results. Is the naming incorrect here 🤔. Or do we want to pass down some simple things such as the username or names. 

If this is just simply for getting all the profiles on the database then perhaps a new name would be more clear.